src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnGitRepository.cs

```diff
@@ -24,7 +23,7 @@ public class AltinnGitRepository : GitRepository, IAltinnGitRepository
     {
         private const string SCHEMA_FILES_PATTERN_JSON = "*.schema.json";
         private const string STUDIO_SETTINGS_FILEPATH = ".altinnstudio/settings.json";
-        private const string TEXT_FILES_PATTERN_JSON = "*.texts.json";
+        private const string TEXT_FILES_PATTERN_JSON = "*.texts.*";
```

Comment:
Why an * at the end? Isn't it *.texts.json ?

Reply:
Wonder about the same 😂